kibana-svc.yaml

```diff
@@ -2,13 +2,12 @@ apiVersion: v1
 kind: Service
 metadata:
   name: kibana
-  namespace: default
   labels:
     component: kibana
 spec:
   selector:
     component: kibana
   ports:
-  - port: 80
-    targetPort: 5601
-    protocol: TCP
+  - name: http
```

Comment:
These changes to kibana-svc.yaml broke the accessing of Kibana via the kubernetes API server proxy for me.

Reply:
can you please elaborate on "broke"? error message? anything?

Reply:
Hitting this URL in Firefox:

  http://127.0.0.1:8443/api/v1/namespaces/default/services/kibana/proxy

yields:

{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {
    
  },
  "status": "Failure",
  "message": "no endpoints available for service \"kibana\"",
  "reason": "ServiceUnavailable",
  "code": 503
}